draft-ietf-quic-transport.md

```diff
@@ -2288,16 +2297,6 @@ local address.  Failure of path validation simply means that the new path is not
 usable for this connection.  Failure to validate a path does not cause the
 connection to end unless there are no valid alternative paths available.
 
-An endpoint uses a new connection ID for probes sent from a new local address;
-see {{migration-linkability}} for further discussion. An endpoint that uses
-a new local address needs to ensure that at least one new connection ID is
-available at the peer. That can be achieved by including a NEW_CONNECTION_ID
-frame in the probe.
```

Comment:
I initially thought this text was essential to retain, but in fact, it stems from the assumption that the peer is required to reply on the new path (and therefore with a fresh CID).  That's not a bad thing for the peer to do and therefore including the NCID frame in a probe is also worth recommending, but it's not as critical as it used to be.

Reply:
Re-reading this, I think I should retain the first sentence.  I'm not sure if I need to retain the second two, given this is only about probing a new path?

Reply:
The first sentence is probably duplicative, but it leads into the second -- if the peer is to respond *on the same path*, they will need an extra CID, so bundling a NCID frame is a good idea.  (That's actually why NCID is allow in probing packets, IIRC.)  But on the other hand, the peer isn't required to respond on the same path any longer, so no harm done if omitted.

Reply:
It might still be worth suggesting including NCID somewhere, even if not in the same place as before, I think.

Reply:
I restored this paragraph and made one tweak to clarify that the reason the peer needs a new CID is if you expect to receive packets on the new local address.  PTAL.

Reply:
And then I found the paragraph that I thought was duplicative up above in Path Validation, so I removed this paragraph and made some adjustments to the one above, most notably adding the first sentence of this paragraph.  PTAL(again)